lib/raven/configuration.rb

```diff
@@ -127,6 +127,12 @@ def initialize
       self.sanitize_fields = []
       self.sanitize_credit_cards = true
       self.environments = []
+
+      self.release = ENV['HEROKU_SLUG_COMMIT']
+
+      if self.release.nil? || self.release.empty?
+        self.release = `git rev-parse --short HEAD`.strip rescue nil
```

Comment:
I guess in that case `release` will be nil. Anything else we can fall back to? I guess not.